### ISSUE TITLE
Get rid of the big image in docs unfurls

### ISSUE THREAD

**Issue by joellabes** [fishtown-analytics/docs.getdbt.com#1039]
### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### What page(s) or areas on docs.getdbt.com are affected?

As far as I can tell, all pages.

![image](https://user-images.githubusercontent.com/7335046/150461217-576d019b-3df6-4bb1-a0da-6a3d332e0ac2.png)

### What changes are you suggesting?

The image unfurls (in Slack etc) for docs are enormous, don't add value and drastically decrease the information density of the screen. I think we should get rid of them altogether

**Comment by runleonarun** [comment]
I think this might be the [image source](https://github.com/dbt-labs/docs.getdbt.com/blob/current/website/static/img/avatar.png). Its source is pretty big. 

I wonder if instead of nixing the image, we should find an intentional design that looks good and more subtle?

**Comment by KiraFuruichi** [comment]
May be an obvious question, but if we nixed the image, would this impact link preview on social (twitter/LI)? Since I think folks are more likely to click on a link that unfurls to something vs just seeing a link (at least on social).

![image](https://user-images.githubusercontent.com/18452766/193625849-f54d02b7-b3c5-4f92-b08c-bd6fe6c45574.png)

**Comment by joellabes** [comment]
@KiraFuruichi yeah this is a really good point! Three thoughts come to mind: 
- Can we remove them just for documentation? That's the majority of what's shared in Slack (and it's more common to share multiple help links, which leads to the overloaded one I shared up top), while keeping the images for posts under the `blog/` path
- I assume all unfurls come from the same metadata, and we can't say "don't show them on Slack, but do show them everywhere else", but if there's any level of config then that'd be great too.
- I agree that images lead to more clickthroughs on social, but I have to imagine that _relevant_ images outperform the same static image. Maybe we need to jazz them up? But that devolves to <details><summary>stock images</summary>
![image](https://user-images.githubusercontent.com/7335046/193652489-d46d27c6-fe4d-4f9b-9245-b86e466b4c09.png)</details> pretty quickly

**Comment by KiraFuruichi** [comment]
TL;DR I'm inclined to remove the images just for documentation, if that's possible!

Your suggestion of showing more relevant images make a ton of sense, but also requires the person making the PR/blog post to remember to add that in or ensure they change the image in Hubspot. Not the end of the world, just an extra layer of complexity. We also potentially run into the issue of using images we're not allowed to, but that's not something I'd be too worried about.

**Comment by lfracalossiFA** [comment]
@joellabes @KiraFuruichi Are we talking about removing images for outward-facing documentation pages? I would highly suggest not doing this -- I suspect it would have a big impact on social media marketing. At the very least I'd run it by @emilymermell-dbt or @krevitt first. 

Also, if you could please check the requirement on social preview images, that would be lovely. I'm not actually sure which one Slack uses -- if it goes off the OG or Twitter meta tags -- but as I recall they do have some specific requirements as to image sizes, content, etc.

Also since this would be a change for my team, and we don't have great visibility into GitHub Issues, could you open an Asana task for this?

**Comment by lfracalossiFA** [comment]
Twitter summary card documentation: https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary

You can use this tool to preview how OG tags will display: https://dnschecker.org/open-graph-preview-generate-metatags.php?query=https%3A%2F%2Fgetdbt.com

**Comment by joellabes** [comment]
Sounds like this has more downstream effects than expected, so let's pause this until post-coalesce - everyone already has enough going on! I'll pick this back up once the dust settles 🙏

**Comment by runleonarun** [comment]
@joellabes I'm closing this as `wontfix` but please reopen if you think it's impactful work.
